src/ontology/config/mappings-to-uberon-bridge.rules

```diff
@@ -0,0 +1,16 @@
+prefix UBERON:    <http://purl.obolibrary.org/obo/UBERON_>
+prefix CL:    <http://purl.obolibrary.org/obo/CL_>
+
+# Make sure UBERON/CL classes, and only UBERON/CL classes, are on the object side
+subject==UBERON:* -> invert();
+!object==UBERON:* -> stop();
+
+# Ignore any mapping to/from an inexistent or obsolete class.
+predicate==* -> check_object_existence();
+predicate==* -> check_subject_existence();
+
+# Ignore any mapping where the same foreign term is mapped to more than one UBERON class.
+!cardinality==*:1 -> stop();
+
+# Create the hasDbXref annotation and annotate it with a subset of the mapping metadata
```

Comment:
I just note that this comment does not match what the rule does. The rule creates a `SubClassOf` axiom, it has nothing to do with a `oboInOwl:hasDbXref` annotation.

Reply:
Copy paste error, fixed now thank you!